### PR TITLE
Add NN MLP Classifier, confusion matrix and balanced acc score

### PR DESCRIPTION
**Pull request by AndreasPB**
* New model
* More tools for analysing performance

### CODE REVIEW

**Comment by AndreasPB** [comment]
Im not 100% sure what I did is Youden Index, but idk

File: src/2_train/util.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.metrics import roc_curve
+from sklearn.metrics import classification_report, balanced_accuracy_score
+from sklearn.metrics import confusion_matrix
+from sklearn import metrics
+
+
+def summary_report(y_test: list, y_test_pred: list, msg: str | None):
```
Comment (Primdahl26) [inline src/2_train/util.py:9]: Python 3.9 syntax pls

**Review by Primdahl26 — CHANGES_REQUESTED** [review]

File: src/2_train/util.py (hunk 1 above)
Comment (AndreasPB) [reply src/2_train/util.py]: Done

**Review by Primdahl26 — APPROVED** [review]

**Pull request merged**